### ISSUE TITLE
Needs update for ARCore 1.2

### ISSUE THREAD

**Issue by pnduffy** [jonas-johansson/ARCoreUtils#8]
I found this library very useful, however it needs an update to support ARCore 1.2 (just released).  The generated mesh needs to work on vertical planes which are now supported in 1.2.  The 'flickering' happens on vertical planes but not the old horizontal ones.

**Comment by ZeroLu** [comment]
Agreed. I believe that it's because that the triangulator only works on the XZ plane.

**Comment by ZeroLu** [comment]
All right. I found that you can simply use the mesh from the DetectedPlaneVisualizer as the collider mesh, which works on vertical planes as well
